lib/registry.js

```diff
@@ -4,21 +4,23 @@ var ReleaseDB = contract(require("../build/contracts/ReleaseDB.json"));
 var semver = require("semver");
 
 function Registry(registry_address, from_address, provider) {
-  PackageIndex.setProvider(provider);
   ReleaseDB.setProvider(provider);
```

Comment:
To be honest I'm not sure I understand the purpose of this ReleaseDB object and setting its provider and defaults.